### PR TITLE
Chemhours

### PR DESCRIPTION
**Pull request by SumilThakr**
Please take a look.

### CODE REVIEW

**Comment by ctessum** [comment]
So, I think the first thing that needs to be done is to rebase your changes so that they are relative to the most recent master version. To do that, you first need to make a copy of the master version (after making a backup copy of what you have):

``` bash
git remote add spatialmodel git@github.com:spatialmodel/inmap.git # https://help.github.com/articles/adding-a-remote/
git fetch -f spatialmodel master # https://git-scm.com/docs/git-fetch
```
n.b. the commands above might overwrite any changes you have made in your `master` branch.

Then, rebase your changes on top of the master branch

``` bash
git checkout chemhours
git rebase master
```

There may be some conflicting changes that you have to manually resolve.

Finally, push the rebased version to the pull request:
``` bash
git push -f origin chemhours
```

**Comment by SumilThakr** [comment]
In the last step, do you mean

git push -f sumil chemhours ?

**Comment by SumilThakr** [comment]
Where the sumil remote is https://github.com/SumilThakr/inmap.git
and the origin remote is the one I don't have permission for:
https://github.com/spatialmodel/inmap.git

**Comment by ctessum** [comment]
The tests are still failing; did you say you had an updated version of the golden file that causes the tests to pass?

File: preproc_test.go [hunk 1]
```diff
@@ -65,6 +65,57 @@ func BenchmarkWRFChemToInMAP(b *testing.B) {
...
+		"cmd/inmap/testdata/preproc/GlobalTestData/GEOSFP.[DATE].A3mstE.2x25.nc",
+		"cmd/inmap/testdata/preproc/GlobalTestData/GEOSFP.ApBp.nc",
+		"cmd/inmap/testdata/preproc/GlobalTestData/ts.[DATE].nc",
+		"cmd/inmap/testdata/preproc/GlobalTestData/vegtype.global.txt",
+		"20150102",
+		"20150103",
+		false,
+		"3h",
+		"24h",
+		true,
```
Comment (ctessum) [inline preproc_test.go]: THIS is the problem

**Comment by ctessum** [comment]
Closing as redundant to #48.

**Pull request closed without merging**